client/blocks/color-scheme-picker/index.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import QueryPreferences from 'components/data/query-preferences';
+import { savePreference, setPreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { ColorSchemes } from './constants';
+import FormRadiosBar from 'components/forms/form-radios-bar';
+
+class ColorSchemePicker extends PureComponent {
+	static propTypes = {
+		temporarySelection: PropTypes.bool,
+		onSelection: PropTypes.func,
+		// Connected props
+		colorSchemePreference: PropTypes.string,
+		saveColorSchemePreference: PropTypes.func,
+	};
+
+	handleColorSchemeSelection = event => {
```

Comment:
FWIW I think that having the parentheses makes it clearer, so I'd +1 making the change :)